portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmailReport.testcase

```diff
@@ -41,6 +41,68 @@ definition {
 		}
 	}
 
+	@description = "Story: LRAC-11664 | Automation ID: LRAC-11844 | Test Summary: Check that cancel button does not enable email report"
+	@priority = "3"
+	test CanCancelChangesToEmailReport {
+		task ("Sign in AC as Test Test user") {
+			ACUtils.launchAC();
+		}
+
+		task ("Go to property and Select the property you are using") {
+			ACNavigation.goToSettings();
+
+			ACSettings.goToProperties();
+
+			ACNavigation.openItem(itemName = "${assignedPropertyName}");
+		}
+
+		task ("Enable and configure email report frequency") {
+			ACEmailReport.accessEmailReportSettings();
+
+			ACEmailReport.enableEmailReportToggle();
+
+			ACUtils.selectGenericOption(optionValue = "Daily");
+
+			ACNavigation.clickCancelButton();
+		}
+
+		task ("Check that email report has been enabled") {
+			ACUtils.viewGenericText(textValueList = "Email Reports: Disabled");
+		}
+	}
+
+	@description = "Story: LRAC-11664 | Automation ID: LRAC-11845 | Test Summary: Check email report edit disabled"
+	@priority = "3"
+	test CheckEmailReportEditDisabled {
+		task ("Sign in AC as Test Test user") {
+			ACUtils.launchAC();
+		}
+
+		task ("Go to property and Select the property you are using") {
+			ACNavigation.goToSettings();
+
+			ACSettings.goToProperties();
+
+			ACNavigation.openItem(itemName = "${assignedPropertyName}");
+		}
+
+		task ("Enable and configure email report frequency") {
+			ACEmailReport.accessEmailReportSettings();
```

Comment:
Hey Calvin, would it be better to rename this task to `Go to email report settings`? because in the way it's, another person reading might believe that you forgot to enable and change the frequency. What do you think?